src/todo/style.lua

```diff
@@ -0,0 +1,45 @@
+local default_gui = data.raw["gui-style"].default
```

Comment:
Nice. I never got around to learn how styles work in Factorio. Thanks for implementing this :)

Reply:
Too many years of front-end web development, finally paying off!